src/ManifoldsBase.jl

```diff
@@ -542,19 +579,40 @@ By default these informations are collected by calling [`check_approx`](@ref).
 Keyword arguments can be used to specify tolerances.
 """
 function isapprox(M::AbstractManifold, p, X, Y; error::Symbol = :none, kwargs...)
-    mat = check_approx(M, p, X, Y; kwargs...)
-    if mat !== nothing
-        (error === :error) && throw(mat)
-        if isnan(mat.val)
-            s = "$(typeof(mat))\n$(mat.msg)"
-        else
-            s = "$(typeof(mat)) with $(mat.val)\n$(mat.msg)"
+    if error === :none
+        return _isapprox(M, p, X, Y; kwargs...)::Bool
+    else
+        mat = check_approx(M, p, X, Y; kwargs...)
+        if mat !== nothing
+            (error === :error) && throw(mat)
+            if isnan(mat.val)
+                s = "$(typeof(mat))\n$(mat.msg)"
+            else
+                s = "$(typeof(mat)) with $(mat.val)\n$(mat.msg)"
+            end
+            (error === :info) && @info s
+            (error === :warn) && @warn s
+            return false
         end
-        (error === :info) && @info s
-        (error === :warn) && @warn s
-        return false
+        return true
     end
-    return true
+end
+
+"""
+    _isapprox(M::AbstractManifold, p, q; kwargs...)
+
+See documentation of `check_approx`.
```

Comment:
If we keep it this way, please with ``[`check_approx`](@ref)``, but I would prefer at least a bit of docs here as well (that it is a faster true/false variant of check).

Reply:
OK, I will expand the docs a bit.